tests/index.html

```diff
@@ -90,7 +94,7 @@
     </script>
 
     <script type="text/javascript">
-      var setTesting = Ember.__loader.require('@ember/debug')['setTesting'];
```

Comment:
Why doesn't this work?

Reply:
I started this branch building on the deprecation work, so this is a change that will need to happen to avoid the deprecation.